helm-charts/falcon-image-analyzer/values.yaml

```diff
@@ -31,6 +31,12 @@ image:
   # private repository
```

Comment:
Add a comment here that `pullSecret` should be a reference to a secret in the same namespace - `falcon-image-anlyzer`